ProjectLighthouse.Servers.GameServer/Controllers/Login/LoginController.cs

```diff
@@ -18,15 +17,8 @@ namespace LBPUnion.ProjectLighthouse.Servers.GameServer.Controllers.Login;
 [ApiController]
 [Route("LITTLEBIGPLANETPS3_XML/login")]
 [Produces("text/xml")]
-public class LoginController : ControllerBase
+public class LoginController(DatabaseContext database) : ControllerBase
 {
-    private readonly DatabaseContext database;
-
-    public LoginController(DatabaseContext database)
-    {
-        this.database = database;
-    }
-
```

Comment:
Didn't we want to hold off on using primary constructors? If I recall correctly there was a PR that we removed those changes from.

Reply:
I don't know, I was just listening to rider. I can revert the change if need be.